mitodl/ocw-studio
# Use common mgmt command options for filtering by websites

#### Pre-Flight checklist



#### What are the relevant tickets?
Closes #1388 
Closes #1135

#### What's this PR do?
Adds a base `WebsiteFilterCommand` class that provides `--filter` and `--filter-json` options.  Modifies numerous existing commands to inherit from this wherever it seems useful.

#### How should this be manually tested?
You can try some or all the modified commands that now inherit from `WesbsiteFilterCommand` to make sure they work correctly with and without the `--filter` option.
